DToCToDB.md

```diff
@@ -0,0 +1,1193 @@
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>14</th>
+      <td>2016-17</td>
+      <td>OCTOBER</td>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Acute</td>
+      <td>E_CARE_PACKAGE_IN_HOME</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>15</th>
+      <td>2016-17</td>
+      <td>OCTOBER</td>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Acute</td>
+      <td>DI_RESIDENTIAL_HOME</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>16</th>
+      <td>2016-17</td>
+      <td>OCTOBER</td>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Acute</td>
+      <td>DII_NURSING_HOME</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>17</th>
+      <td>2016-17</td>
+      <td>OCTOBER</td>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Acute</td>
+      <td>C_FURTHER_NON_ACUTE_NHS</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>18</th>
+      <td>2016-17</td>
+      <td>OCTOBER</td>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Acute</td>
+      <td>B_PUBLIC_FUNDING</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <th>19</th>
+      <td>2016-17</td>
+      <td>OCTOBER</td>
+      <td>Q82</td>
+      <td>NHS ENGLAND SOUTH (SOUTH CENTRAL)</td>
+      <td>AXG</td>
+      <td>WILTSHIRE HEALTH &amp; CARE LLP</td>
+      <td>817</td>
+      <td>WILTSHIRE</td>
+      <td>Acute</td>
+      <td>A_COMPLETION_ASSESSMENT</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+      <td>0</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+The aim of this exercise is to combine all the available datasets into a single dataset, to then insert into a SQLite table within a database that contains other NHS datasets.
```

Comment:
Explaining the above columns would be helpful (depending on who your audience for this notebook is)